### PR TITLE
httpry: add page

### PR DESCRIPTION
**Pull request by WhileLoop**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by WhileLoop** [comment]
Closes #4095.

File: pages/common/httpry.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# httpry
+
+> A lightweight packet sniffer for displaying and logging HTTP traffic.
+> It can be run in real-time displaying the traffic as it is parsed, or as a daemon process that logs to an output file.
+> More information: http://dumpsterventures.com/jason/httpry/.
```
Comment (navarroaxel) [inline pages/common/httpry.md:5]:
```suggestion
> More information: <http://dumpsterventures.com/jason/httpry/>.
```

File: pages/common/httpry.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Save output to a file:
+
+`httpry -o out.log`
```
Comment (navarroaxel) [inline pages/common/httpry.md:9]:
```suggestion
`httpry -o {{path/to/file.log}}`
```

File: pages/common/httpry.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- Listen on a specific interface and save output to a binary pcap format file:
+
+`httpry -i eth0 -b out.pcap`
```
Comment (navarroaxel) [inline pages/common/httpry.md:13]:
```suggestion
`httpry -i {{eth0}} -b {{path/to/file.pcap}}`
```

File: pages/common/httpry.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- Filter output by HTTP method strings:
```
Comment (navarroaxel) [inline pages/common/httpry.md:15]: :thinking: I always used `HTTP verbs` but MDN call them `request methods` 
```suggestion
- Filter output by a comma-separated list of HTTP method strings:
```

File: pages/common/httpry.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+`httpry -m get,head`
```
Comment (navarroaxel) [inline pages/common/httpry.md:17]: We can help with the option list here:
```suggestion
`httpry -m {{get|post|put|head|options|delete|trace|connect|patch}}`
```

File: pages/common/httpry.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+- Read from an input capture file and filter by IP:
+
+`httpry -r out.log 'host 192.168.5.25'`
```
Comment (navarroaxel) [inline pages/common/httpry.md:21]: Are you sure the `bpf-style` expression is not `src host ip`? based on reading the github README.
```suggestion
`httpry -r {{path/to/file.log}} '{{host 192.168.5.25}}'`
```

File: pages/common/httpry.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+- Run as daemon process:
+
+`httpry -d -o /var/log/http.log`
```
Comment (navarroaxel) [inline pages/common/httpry.md:25]:
```suggestion
`httpry -d -o {{path/to/file.log}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @WhileLoop I left some comments.
WhileLoop w

**Comment by WhileLoop** [comment]
@navarroaxel Thanks for the feedback. I believe I have addressed all the issues.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @WhileLoop :slightly_smiling_face:

**Comment by bl-ue** [comment]
Oops! It looks like this didn't close the issue references because it wasn't in the description.  I'll close it myself.

**Pull request merged**